### PR TITLE
error analysis + update prepro gdirs

### PR DESCRIPTION
**Pull request by lilianschuster**
- added further explanation about the different pre-processed gdirs inside of `input-data.rst`
  - specifically about the MB calibration type and whether quality check is included or not (because this was asked in the Slack channel)
- corrected some references and links
- added an [error_analysis notebook](https://nbviewer.org/urls/cluster.klima.uni-bremen.de/~lschuster/error_analysis/error_analysis_v1.ipynb) inside of `input-data.rst` together with one summary plot that is now also inside of the documentation
  - If needed, I can add the bullet-point list from the notebook as well into the documentation. But maybe it is sufficient, when it is only in the notebook.
 
I am not sure if the formatting of e.g. the :func: and :ref: worked as it should. I guess, I can check this only when it is already online? 

- [x] Fully documented

### CODE REVIEW

File: docs/input-data.rst [hunk 1]
```diff
@@ -239,21 +242,62 @@ flowlines:
...
+
+There are different mass-balance calibration options available in the preprocessed directories:
+
+-   **no_match**: This is the default calibration option. For calibration, the direct glaciological WGMS data is used
+    and the `Marzeion et al., 2012`_ *tstar* method is applied to interpolate to glaciers without measurements
+    (see also :ref:`mass-balance-2012`). With this method, the geodetic estimates are not matched.
+-   **match_geod**: The default calibration with direct glaciological WGMS data is still applied on the glacier per
+    glacier level, but on the regional level the epsilon (residual) is corrected to match the geodetic estimates
+    from `Hugonnet et al., 2021`_ (using :py:func:`oggm.workflow.match_regional_geodetic_mb`, see also
+    [regional calibration](https://docs.oggm.org/en/latest/mass-balance-2012.html)). For example:
```
Comment (fmaussion) [inline docs/input-data.rst:260]: Replace `[regional calibration](https://docs.oggm.org/en/latest/mass-balance-2012.html)` with a proper sphinx reference

**Comment by fmaussion** [comment]
Great thanks! Only one tiny comment.

**Comment by lilianschuster** [comment]
- added this notebook: https://nbviewer.org/urls/cluster.klima.uni-bremen.de/~lschuster/error_analysis/working_glacier_gdirs_comparison.ipynb?flush_cache=true
- added documentation to the geodetic convert notebook: https://nbviewer.org/urls/cluster.klima.uni-bremen.de/~oggm/geodetic_ref_mb/convert.ipynb 
- included everywhere ?flush_cache=true inside of the nbviewers in order that they are always updated
- improved the internal links as suggested 
- added remarks that match_geod_pergla makes only sense with qc0 and that we will soon remove all  match_geod_pergla  qc3 (when we have created new match_geod_pergla  qc0 gdirs )
- if this is interesting to add, could also link this notebook to the documentation which shows differences for future projections: https://nbviewer.org/urls/cluster.klima.uni-bremen.de/~lschuster/error_analysis/vas_error_analysis.ipynb#id-volume-proj-single-gcm?flush_cache=true

File: docs/contributing.rst [hunk 2]
```diff
@@ -210,18 +210,19 @@ Requirements
 ~~~~~~~~~~~~
 
 There are some extra requirements to build the docs: you will need to
-have ``sphinx``, ``sphinx_rtd_theme``, ``numpydoc`` and ``ipython`` installed.
+have ``sphinx``, ``sphinx_rtd_theme``, ``sphinx-togglebutton``, ``sphinx-book-theme``, ``numpydoc``
```
Comment (fmaussion) [inline docs/contributing.rst:213]:
```suggestion
have ``sphinx``,  ``sphinx-togglebutton``, ``sphinx-book-theme``, ``numpydoc``
```

File: docs/contributing.rst [hunk 3]
```diff
@@ -210,18 +210,19 @@ Requirements
...
+and ``ipython`` installed.
 
 If you have a conda environment named ``oggm_env``, you can install the extra
 requirements with::
 
       conda install -n oggm_env sphinx sphinx_rtd_theme ipython numpydoc
```
Comment (fmaussion) [inline docs/contributing.rst:219]:
```suggestion
      conda install -n oggm_env ipython sphinx numpydoc
```

File: docs/contributing.rst [hunk 4]
```diff
@@ -210,18 +210,19 @@ Requirements
...
+      conda install -c conda-forge sphinx-togglebutton sphinx-book-theme
 
 If you use pip, activate your python environment and install the requirements
 with::
 
-      pip install sphinx sphinx_rtd_theme ipython numpydoc
-
+      pip install sphinx sphinx_rtd_theme sphinx-togglebutton sphinx-book-theme ipython numpydoc
```
Comment (fmaussion) [inline docs/contributing.rst:225]:
```suggestion
      pip install sphinx sphinx-togglebutton sphinx-book-theme ipython numpydoc
```

**Review by fmaussion — APPROVED** [review]
Fantastic! Can you remove all references to `sphinx_rtd_theme`? This is not needed anymore

**Comment by lilianschuster** [comment]
- now, also all `sphinx_rtd_theme`  references are removed!

**Comment by fmaussion** [comment]
Thanks Lily!

**Pull request merged**